test/hotspot/jtreg/runtime/CompressedOops/CompressedClassPointers.java

```diff
@@ -98,7 +98,12 @@ public static void largeHeapTest() throws Exception {
             "-Xshare:off",
             "-XX:+VerifyBeforeGC", "-version");
         OutputAnalyzer output = new OutputAnalyzer(pb.start());
-        if (testNarrowKlassBase()) {
+        if (testNarrowKlassBase() && !Platform.isAix()) {
+            // AIX: the heap cannot be placed below 32g. The first attempt to
+            // place the CCS after the heap fails (luckily). Finally CCS is
+            // successfully placed below 32g. So we get 0x0 as narrow klass
+            // base.  As an enhancement the first attempt mapping the CCS should
+            // be made below 32g if oops are compressed but the heap is above 32g.
```

Comment:
I'd remove the enhancement remark since it is slightly confusing and irrelevant to the test in its current form. If you want to mention it, could you please create a JBS issue and mention it here ("see also JDK-xxxx").

Reply:
I removed it. I'll create the JBS item after some initial tests.